Samples/360VideoPlayback/cpp/360VideoPlayback.vcxproj

```diff
@@ -217,18 +217,30 @@
       <ShaderModel Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">5.0</ShaderModel>
       <ShaderModel Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">5.0</ShaderModel>
       <ShaderType Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">Geometry</ShaderType>
+      <ShaderType Condition="'$(Configuration)|$(Platform)'=='Release|x64'">Geometry</ShaderType>
```

Comment:
Were these changes intended to be part of the pull request?

Reply:
Yes. As it turns out, the sample would fail to build on Release configurations, preventing me from fully testing the fix.